rules/anti-static/obfuscation/bitwise.yara

```diff
@@ -188,6 +188,17 @@ rule bitwise_obfuscation: critical {
     $f_substr  = "substr("
     $f_ord     = "ord("
 
+    $not_phpseclib1 = "http://phpseclib.sourceforge.net"
```

Comment:
Do share my perspective that this might be a losing strategy no matter what when we're trying to catch obfuscation as a signal? Bitwise operations and paddings seem really hard to key on and we'll have to keep updating exceptions. 

For me I'd like some kind of confidence score for each rule that gets fed into the decision around blocking/no blocking and these exceptions could improve our confidence score. 

This for example is an item that by itself would be low confidence but mixed with other signal might result in something higher. 

Short term: I feel this is high noise and therefore can't be a blocker. 
Long term: Something that will allow us to tweak the implementation of the rulesets for our environment, without the criticality score.